src/plugins/navigation/review/navigatorFizzy.tpl

```diff
@@ -1,21 +1,12 @@
-<div class="qti-panel qti-navigator{{#if hidden}} hidden{{/if}} fizzy">
-    <span class="qti-navigator-label" title="{{__ 'Test overview'}}">
-            <span class="qti-navigator-text">{{__ 'Test overview'}}</span>
-            <a class="icon-close" href="#" onclick="return false" aria-hidden="true"></a>
-    </span>
-
-    <div class="qti-navigator-filters">
-        <div class="qti-navigator-filter active">
-            <nav class="qti-navigator-tree"></nav>
-        </div>
+<div class="qti-panel qti-navigator{{#if hidden}} hidden{{/if}} qti-navigator-fizzy">
```

Comment:
I found one case where a previous style was lost:
#### old:
![Screenshot 2022-02-22 at 09 41 30](https://user-images.githubusercontent.com/43652944/155094988-8b699ba9-9839-4c52-8afa-6535b562fd3d.png)
#### new:
![Screenshot 2022-02-22 at 09 41 38](https://user-images.githubusercontent.com/43652944/155095020-ba93d150-e278-4798-90e9-bb2fd964cccf.png)

Reply:
I'll leave left/right margin - for me it looks prettier, plus I don't wan to override parent margin on `.qti-panel`. There's no mockup for this case anyway.